test/battle/ability/gale_wings.c

```diff
@@ -0,0 +1,47 @@
+#include "global.h"
+#include "test/battle.h"
+
+SINGLE_BATTLE_TEST("Gale Wings only grants priority at full HP")
+{
+    u16 hp;
+    PARAMETRIZE { hp = 100; }
+    PARAMETRIZE { hp = 99; }
+    GIVEN {
+        ASSUME(B_GALE_WINGS >= GEN_7);
+        ASSUME(gBattleMoves[MOVE_AERIAL_ACE].type == TYPE_FLYING);
+        PLAYER(SPECIES_TALONFLAME) { Ability(ABILITY_GALE_WINGS); HP(hp); MaxHP(100); Speed(1);}
+        OPPONENT(SPECIES_WOBBUFFET) { Speed(100);};
+    } WHEN {
+        TURN { MOVE(player, MOVE_AERIAL_ACE); }
+    } SCENE {
+        if (hp == 100) {
+            MESSAGE("Talonflame used Aerial Ace!");
+        }
+        else {
+            MESSAGE("Foe Wobbuffet used Celebrate!");
+        }
```

Comment:
This doesn't properly show that one message happens after the other.
A single `MESSAGE` just checks if the message happens at all.
```suggestion
        if (hp == 100) {
            MESSAGE("Talonflame used Aerial Ace!");
            MESSAGE("Foe Wobbuffet used Celebrate!");
        }
        else {
            MESSAGE("Talonflame used Aerial Ace!");
            MESSAGE("Foe Wobbuffet used Celebrate!");
        }
```